docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md

```diff
@@ -105,26 +111,37 @@ That should provoke logging like the following from the mailbox controller.
 I0305 18:07:20.490417   85556 main.go:369] "Created missing workspace" worker=0 mbwsName="niqdko2g2pwoadfb-mb-f99e773f-3db2-439e-8054-827c4ac55368"
 ```
 
-And you can verify that like so.
+And you can verify that as follows:
 
 ```console
-$ kubectl ws root:edge
-Current workspace is "root:edge".
+$ kubectl ws root:espw
+Current workspace is "root:espw".
 
-$ kubectl get Workspace
+$ kubectl get workspace
```

Comment:
`Workspace` is the "kind", `workspaces` is the "resource"; `workspace` is nothing but a convenience for sloppiness, not something we should recommend.

Reply:
Thanks, I meant to use the resource (`workspaces`) - fixed in latest push.